sos/report/plugins/openstack_keystone.py

```diff
@@ -106,6 +112,10 @@ def postproc(self):
             "|".join(connection_keys),
             r"\1*********\6"
         )
+        self.apply_regex_sub(
+            r"(^\s*(%s)\s*)(.*)" % "|".join(client_secrets),
+            r"\1*********"
+        )
```

Comment:
The secret can be in either `/etc/keystone` config or in `httpd` config, such that you call `apply_regex_sub` and not `do_path_regex_sub`, right?

Reply:
That was my understanding from the report and the doc, but let me doublecheck.